components/loading-box/src/index.js

```diff
@@ -8,11 +8,13 @@ import { BLACK, WHITE } from 'govuk-colours';
 
 const spinnerClassName = 'icon-loading';
 
-const StyledContainer = styled('div')({
+const StyledContainer = styled('div')(({
+  loading,
+}) => ({
   position: 'relative',
   paddingBottom: '2px',
-  minHeight: '100px',
-});
+  minHeight: loading ? '100px' : undefined,
```

Comment:
cool - i did think of this... but i don't think it's the right solution here

we have use-cases where the loading box is wrapped around a drop-down menu where this will force the container to be too tall for the duration of when the data is loading

IMHO we're better off either removing this min-height, and relying on the component that's using the loading box to specify a min-height on the contents, or setting it to a much smaller value like 10px as a fall-back "make sure we get to display *something*"